.github/pull_request_template.md

```diff
@@ -19,6 +19,13 @@ Make sure you've run and fixed any issues with these commands:
 - `cargo clippy --workspace -- -D warnings -D clippy::unwrap_used -A clippy::needless_collect` to check that you're using the standard code style
 - `cargo test --workspace` to check that all tests pass
 
+> **Note**
+> you can also use the `toolkit` as follows
```

Comment:
```suggestion
> from nushell you can also use the `toolkit` as follows
```

Reply:
of course, i'm so used to being in `nushell` already :eyes: 
thanks for the reminder :relieved: 

should be fixed in f0d84b0f335079a4f1f61db3a2f66da78a16b3c9 :+1: